gdk/commands/component/InitCommand.py

```diff
@@ -0,0 +1,135 @@
+import logging
+import shutil
+import tempfile
+import zipfile
+from io import BytesIO
+from pathlib import Path
+
+import gdk.commands.component.list as list
+import gdk.common.consts as consts
+import gdk.common.exceptions.error_messages as error_messages
+import gdk.common.utils as utils
+import requests
+from gdk.commands.Command import Command
+
+
+class InitCommand(Command):
+    def __init__(self, command_args, level_command) -> None:
```

Comment:
why `level_command` is required as arg here? `InitCommand` should always have "Init" as `level_command`. So instead we can call super's init with "Init" here directly